src/main/scala/edu/berkeley/ce/rockslicing/BlockVTK.scala

```diff
@@ -36,18 +36,18 @@ object BlockVTK {
     // vertical line running through this center.
     // Check if points are on opposite sides of the center. Points left of center will be before points
     // right of the center
-    if ((pointA(0) - center(0) < -NumericUtils.EPSILON) && (pointB(0) - center(0) >= NumericUtils.EPSILON)) {
```

Comment:
I think what's important here is where the points lie relative to 0.0. Since this compares two points to see on which side of zero they lie, it becomes important to compare up to zero for very small blocks. If one point is greater than `-EPSILON` but less than 0.0 and the other is just less than `+EPSILON` but greater than 0.0, the ordering of the points should account for that.
This really only becomes and issue when dealing with extremely small blocks. Does that make sense?